pygfx/renderers/wgpu/_renderer.py

```diff
@@ -226,12 +226,13 @@ def blend_mode(self):
         * "simple2": two-pass approach that first processes all opaque fragments and
           then blends transparent fragments (using the OVER operator) with depth-write disabled.
           Yields visually ok results, but is not order independent.
-        * "blended": two-pass approach that yields order independent transparency.
-          This is like "weighted" but without the depth weights. Performance is the same.
-        * "weighted: two-pass approach that yields order independent
-          transparency, with depth weighting (McGuire 2013). The depth range affects the
-          (quality of the) visual result.
-        * "multilayer2": todo 2-layer MLAB + weighted for the rest?
+        * "weighted": two-pass approach that yields order independent transparency, using alpha weights.
+        * "weighted_depth": two-pass approach that yields order independent
+          transparency, with weights based on alpha and depth (McGuire 2013). Note that the depth
+          range affects the (quality of the) visual result.
+        * "weighted_plus": three-pass approach that yields order independent transparency,
+          in wich the front-most transparent layer is rendered correctly, while transparent
+          layers behind it are blended using "weighted".
```

Comment:
No need to let it hold up this PR, but any suggestions for better names are welcome :)